honeybee_radiance/modifier/material/metal2.py

```diff
@@ -11,7 +11,6 @@ class Metal2(Material):
 
     Metal2 is the same as plastic2, except that the highlights are modified by the
     material color.
-
```

Comment:
Is there a reason why you didn't implement `Args` and `Properties` for this class?

Reply:
Actually, these classes not implemented, in my jugement, weren't supposed to have Args/Properties, As I mentioned above, I just forgot to deleted them all!